src/app/views/login/SignInForm.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import Input from "../../components/Input";
+
+const propTypes = {
+    formData: PropTypes.shape({
+        inputs: PropTypes.arrayOf(
+            PropTypes.shape({
+                id: PropTypes.string,
+                label: PropTypes.string,
+                type: PropTypes.string,
+                onChange: PropTypes.func,
+                error: PropTypes.string
+            })
+        ),
+        onSubmit: PropTypes.func,
+        isSubmitting: PropTypes.bool
+    })
+};
+
+export default class LoginForm extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const inputs = this.props.formData.inputs;
+        const isSubmitting = this.props.formData.isSubmitting;
+
+        return (
+            <div className="grid grid--justify-center">
+                <div className="grid__col-3">
+                    <h1 className="line-height--46">Log in to your Florence account</h1>
```

Comment:
As per the description, this is mostly a duplicate file that will be worked on in future PRs.

- This line however is changed.